content/docs/3_reference/3_panel/3_fields/0_files/reference-article.txt

```diff
@@ -4,7 +4,7 @@ Title: Files
 
 Intro:
 
-A files select field that allows to select one or multiple related files
+A 'files' 'select' field that allows selecting one or multiple related files
```

Comment:
These two quoted words seem weird. And we are not using single quotes anywhere else for input fields, so I wouldn't start this here, either.